helm-chart/banzai/templates/_helpers.tpl

```diff
@@ -87,62 +90,63 @@ Define shared environment variables
   value: {{ .Values.postgresql.postgresqlDatabase | quote }}
 - name: DB_ADDRESS
   value: postgres://$(DB_USER):$(DB_PASSWORD)@$(DB_HOST)/$(DB_NAME)
+{{ else }}
+- name: DB_ADDRESS
+  value: {{ .Values.databaseUrl | quote }}
+{{- end }}
+{{- if .Values.useDockerizedRabbitMQ }}
+- name: RABBITMQ_HOST
+  value: {{ include "banzai.rabbitmq" . | quote }}
 - name: RABBITMQ_PASSWORD
   valueFrom:
     secretKeyRef:
       name: banzai-secrets
       key: rabbitmq-password
 - name: TASK_HOST
   value: amqp://{{ .Values.rabbitmq.rabbitmq.username }}:$(RABBITMQ_PASSWORD)@$(RABBITMQ_HOST)/{{ .Values.rabbitmq.vhost }}
+{{- else }}
+- name: TASK_HOST
+  value: {{ .Values.taskHost | quote}}
+{{- end }}
 - name: RETRY_DELAY
-  value: "600000"
+  value: "600"
+- name: OPENTSDB_PORT
+  value: "80"
 - name: CALIBRATE_PROPOSAL_ID
-  value: {{ .Values.CALIBRATE_PROPOSAL_ID | quote }}
+  value: {{ .Values.calibrateProposalId | quote }}
 - name: OBSERVATION_PORTAL_URL
-  value: {{ .Values.OBSERVATION_PORTAL_URL | quote }}
+  value: {{ .Values.observationPortalUrl | quote }}
 - name: API_ROOT
-  value:  {{ .Values.API_ROOT | quote }}
+  value: {{ .Values.apiRoot | quote }}
 - name: AUTH_TOKEN
-  valueFrom:
-    secretKeyRef:
-      name: banzai-secrets
-      key: AUTH_TOKEN
+  value: {{ .Values.archiveAuthToken | quote }}
+- name: RAW_DATA_FRAME_URL
+  value: {{ .Values.rawDataFrameUrl | quote }}
+- name: RAW_DATA_AUTH_TOKEN
+  value: {{ .Values.rawDataAuthToken | quote }}
 - name: BUCKET
-  value: {{ .Values.BUCKET | quote }}
+  value: {{ .Values.s3Bucket | quote }}
 - name: AWS_ACCESS_KEY_ID
-  valueFrom:
-    secretKeyRef:
-      name: banzai-secrets
-      key: AWS_ACCESS_KEY_ID
+  value: {{ .Values.awsAccessKeyId | quote }}
 - name: AWS_SECRET_ACCESS_KEY
-  valueFrom:
-    secretKeyRef:
-      name: banzai-secrets
-      key: AWS_SECRET_ACCESS_KEY
+  value: {{ .Values.awsSecretAccessKey | quote }}
```

Comment:
Why did we move away from using keys?

Reply:
Discussed over Zoom. It may make sense to re-visit this when and if we decide if we want to provide a helm chart to the public.